packages/arb-token-bridge-ui/src/util/fastBridges.ts

```diff
@@ -77,6 +81,8 @@ export function getFastBridges({
         return `https://stargate.finance/transfer?srcChain=${chainIdToSlug(
           from
         )}&dstChain=${chainIdToSlug(to)}&srcToken=${tokenSymbol}`
+      case FastBridgeNames.deBridge:
+        return `https://app.debridge.finance/?inputChain=${from}&outputChain=${to}&amount=${amount}&inputCurrency=${tokenAddress}`
```

Comment:
if `tokenAddress` is `undefined`, i find that the autofill on the link only works if `tokenAddress` is an empty string, otherwise the amount will not autofill, so we should use `tokenAddress ?? ""`